### ISSUE TITLE
Name property is not provided to the combo-box present throughout the application.: A11y_MSSQL for VSCode_User can create New Table_Add/delete Column_Devtools

### ISSUE THREAD

**Issue by ganagdhar123** [microsoft/vscode-mssql#19813]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS;#MSSQL for VSCode;#SH_MSSQLForVSCode_Win32_Jul2025;#DesktopApp;#FTP;#Win11;#A11ySev2;#WCAG4.1.2;#A11yMAS;
### Environment Details:
Application: MSSQL for VSCode
Visual studio code version: 1.102.1
OS: Windows 11 Enterprise 24H2
Build: 26100.4652
Screen reader
NVDA Version: 2024.4.1
### Repro Steps:

1. Launch visual studio code application
2. Sign in with "GitHub" account
3. Tab till "Extension" and press enter
4. Tab till search box and enter "SQL" (mssql) server and install extension
5. Tab till any database and click on "edit"or "add" and press enter
6. New table screen will get open.
7. Tab till "Column" tab and press enter.
8. Observe that Name property is not provided to the combo-box present throughout the application

### Actual:
Combo-boxes across the application do not have a Name property. As a result, screen readers like NVDA or Narrator do not announce what the combo-box is for, making it inaccessible.
Note: Issue is observed in throughout the application 
### Expected:
Scheme combo-box should have a descriptive Name property exposed to assistive technologies like screen readers. This allows users to understand the purpose of the control.
### User Impact:
Screen reader users are unable to identify the function of combo-boxes, leading to confusion and potential errors in form completion or navigation. This significantly reduces usability and inclusivity for users with visual impairments.
### Attachment:

<img width="1365" height="720" alt="Image" src="https://github.com/user-attachments/assets/3d0c4ef5-00f4-4901-8e34-a4f952cf38c0" />
